File: iter2/iter2eren/main/Classroom.java
package main;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Classroom {

    @JsonProperty("id")
    private String id;

    @JsonProperty("capacity")
    private int capacity;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

}

File: main/Classroom.py



class Classroom :
    __id : str
    __capacity : int

    def __init__(self, id : str, capacity : int):
        self.__id = id
        self.__capacity = capacity

    def getId(self):
        return self.__id

    def setId(self, id : str):
        self.__id = id

    def getCapacity(self):
        return self.__capacity

    def setCapacity(self, capacity : int):
        self.__capacity = capacity

    def to_dict(self):
        return {
            "id": self.__id,
            "capacity": self.__capacity
        }
    
    def toDict(self):
        return {
            "id": self.__id,
            "capacity": self.__capacity
        }